README.md

```
@@ -1,3 +1,65 @@
+# 상품 목록 2단계
+
+담기의 수량을 조절할수있고 장바구니에 담은 목록을 모달로 확인할수있다.
+
+##기능 요구사항
```

markdownlint-cli2 (0.17.2)

No space after hash on atx style heading
null

(MD018, no-missing-space-atx)